js/SearchResult.js

```diff
@@ -0,0 +1,105 @@
+/* search result class */
+class SearchResult {
+  /* search result class constructor */
+  constructor(resultElement) {
+    this.resultList = resultElement;
+  }
+
+  highlightText(company, searchValue, searchKey, elementToAppend) {
```

Comment:
nice implementation of milestone 7, but in the browswer it looks like you have a little bug - sometimes the relevant text is not highlighted.